zipline/finance/performance/position_tracker.py

```diff
@@ -350,14 +350,16 @@ def get_positions(self):
                     pass
                 continue
 
-            # Note that this will create a position if we don't currently have
-            # an entry
-            position = positions[sid]
+            position = zp.Position(sid)
```

Comment:
Not in the docs, but not inconceivable that an algo might use it. So leaving it for now